rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/parametric/ParametricTextDocumentService.java

```diff
@@ -245,8 +252,13 @@ private void invalidateFacts(TextDocumentIdentifier doc) {
     public void didSave(DidSaveTextDocumentParams params) {
         logger.debug("Save: {}", params.getTextDocument());
         // on save we don't get new file contents, that already came in via didChange
-        // but we do trigger the type checker on save
-        triggerSummary(params.getTextDocument());
+        // but we do trigger the type checker on save (if a builder exists)
+        ILanguageContributions contrib = contributions(params.getTextDocument());
+        contrib.hasBuild().thenAccept(b -> { // TODO: `b` can probably be cached
```

Comment:
I think logic needs to move to the `ParametricFileFacts`?

Reply:
Fixed (removed logic as it's actually unnecessary here)